Examples/test-suite/typemap_ns_using.i

```diff
@@ -1,6 +1,9 @@
 %module typemap_ns_using
 
 %warnfilter(SWIGWARN_RUBY_WRONG_NAME) X::_FooImpl;	/* Ruby, wrong class name */
+#ifdef SWIGFORTRAN
+%rename(FooImpl) X::_FooImpl;   /* Fortran CANNOT have class names with leading underscores */
```

Comment:
Change this to a %warnfilter like Ruby please (Fortran should automatically rename to a valid identifier).
Similarly for:
Examples/test-suite/using1.i
Examples/test-suite/using2.i
Examples/test-suite/cpp_basic.i

Reply:
See the discussion on identifier renaming.